nuclear/tools/module.py

```diff
@@ -173,3 +183,74 @@ def generate_test(parts):
     return template.format()
 
 
+def module_unused(**kwargs):
+    rolesPath = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', '..', 'roles'))
+    modulesPath = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', '..', 'module'))
+    basePath = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', '..', 'module')) + os.sep
```

Comment:
Dont use relative paths! Make use of the `cmake_cache`, thats what it is there for!